packages/types/src/chain-adapters/cosmos.ts

```diff
@@ -3,9 +3,51 @@ export type BuildTxInput = {
   fee: string
 }
 
+export type Validator = {
+  address: string
+}
+
+export type Delegation = {
+  assetId: string
+  amount: string
+  validator: Validator
+}
+
+export type RedelegationEntry = {
+  assetId: string
+  completionTime: number
+  amount: string
+}
+
+export type Redelegation = {
+  destinationValidator: Validator
+  sourceValidator: Validator
+  entries: Array<RedelegationEntry>
+}
+
+export type UndelegationEntry = {
+  assetId: string
+  completionTime: number
+  amount: string
+}
+
+export type Undelegation = {
+  validator: Validator
+  entries: Array<UndelegationEntry>
+}
+
+export type Reward = {
+  assetId: string
+  amount: string
+}
+
 export type Account = {
   sequence: string
   accountNumber: string
+  delegations?: Array<Delegation>
+  redelegations?: Array<Redelegation>
+  undelegations?: Array<Undelegation>
+  rewards?: Array<Reward>
```

Comment:
Can we make these required and if there isn't any just return an empty array?

Makes it easier down the line to only deal with an array type instead of `[] | undefined`